ntp/README.md

```diff
@@ -27,12 +27,37 @@ The NTP check is included in the [Datadog Agent][1] package, so you don't need t
 
 ### Configuration
 
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Host" xxx -->
+
+#### Host
```

Comment:
```suggestion
```
I don't think you need these headings, as they appear twice on 
![Screenshot 2024-11-18 at 11 47 31 AM](https://github.com/user-attachments/assets/3feeed31-25e5-4390-aa91-128b34a72f88)
the doc

Reply:
Thanks, @hestonhoffman ! I added them in when I was going through the other examples of these tabs being implemented, I saw that we almost always follow it with the header, so I thought it was some sort of standard we had.

I think the tricky bit is that the `Containerized` tab has a `Metric collection` heading in it too, so that would be the only heading in the tabs remaining. Probably just overthinking it and that's fine?

Reply:
You know what, disregard this edit. I forgot that these tabs are stripped from the tile in the Datadog UI, which is the reason they're repeated elsewhere.

Reply:
Oh TIL! Thanks, Heston!